front-end/src/app/shared/resolvers/transaction.resolver.ts

```diff
@@ -102,26 +111,26 @@ export class TransactionResolver {
     return of(transaction);
   }
 
-  resolveNewChildTransaction(
-    parentTransactionId: string,
-    childTransactionTypeName: string
-  ): Observable<Transaction | undefined> {
-    return this.transactionService.get(String(parentTransactionId)).pipe(
-      mergeMap((parentTransaction: Transaction) => {
-        // If there is a grandparent transaction, then we need to retrieve it
-        if (parentTransaction.parent_transaction_id) {
-          return this.transactionService.get(parentTransaction.parent_transaction_id).pipe(
-            map((grandparent) => {
-              parentTransaction.parent_transaction = grandparent;
-              return this.getNewChildTransaction(parentTransaction, childTransactionTypeName);
-            })
-          );
-        }
-        // Otherwise we just need to return an observable of the parent transaction
-        return of(this.getNewChildTransaction(parentTransaction, childTransactionTypeName));
-      })
-    );
-  }
+  // resolveNewChildTransaction(
```

Comment:
Can this block be removed? Stolen from https://sonarcloud.io/project/issues?id=fecgov_fecfile-web-app&pullRequest=1635&resolved=false&inNewCodePeriod=true

Reply:
There are a few blocks of commented out code - if they can be removed, I think it would improve readability. Thanks @sasha-dresden!

Reply:
Yea I can remove those. Those blocks were to be used for dealing with reattribution or redesignation of transactions on past reports, which will be handled on future tickets, #624 and #1064